core/src/test/java/com/yahoo/imapnio/async/request/UidStoreFlagsCommandTest.java

```diff
@@ -24,6 +25,9 @@
     /** Fields to check for cleanup. */
     private Set<Field> fieldsToCheck;
 
+    /** Unchanged since the known modification sequence */
+    private static final UnchangedSince MOD_SEQ = new UnchangedSince(1L);
```

Comment:
Shouldn't use constant for new created class. If new create class throw exception, all the remaining test cases will not run.